sbirez/management/commands/importtopics.py

```diff
@@ -27,15 +27,15 @@ def handle(self, mdb_filename, solicitation_name, clear=False, **options):
             raise OSError("MS Access database %s not found" % mdb_filename)
 
         # Confirm that solicitation_name is for solicitation in the database
-        proc = subprocess.Popen(#'ls',
```

Comment:
Thanks!  Really, now that I think of it, it should be possible (and would be cleaner) to go through Django's ORM... maybe I should do that today.  Maybe when I look I'll remember why I didn't do that in the first place.